model/task_lifecycle.go

```diff
@@ -96,13 +96,6 @@ func SetActiveState(t *task.Task, caller string, active bool) error {
 		}
 
 		if t.Requester == evergreen.MergeTestRequester {
-			projRef, err := FindOneProjectRef(t.Project)
```

Comment:
Why did these checks get removed instead of changing the method name to FindBranchProjectRef here?